src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -145,6 +147,15 @@ function BaseVideoPlayer({
 
             if (event.fullscreenUpdate === VideoFullscreenUpdate.PLAYER_DID_DISMISS) {
                 isFullScreenRef.current = false;
+
+                currentVideoPlayerRef.current?.getStatusAsync?.().then((status) => {
+                    if (!('isMuted' in status)) {
```

Comment:
Please add a comment here as to why we added this line of code.

Reply:
@shahinyan11 The comment looks good, but please update the comment to `Synchronize the volume settings when switching from full screen to mini-player mode.` We can later confirm with the internal engineer.

Reply:
And why are we returning early if the `isMuted` is not in the `status`?

Reply:
For hiding Typescript error